### PR TITLE
Remove duplicate page of `File.prototype.type`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

`{{DOMxRef("Blob.size")}}` is incorrectly linked to `{{DOMxRef("Blob.prototype.size")}}`

`{{DOMxRef("Blob.prototype.type")}}` is incorrectly linked to `{{DOMxRef("File.prototype.type")}}`

duplicate page of `File.prototype.type` with `Blob.prototype.type`

### Motivation

remove the page of `File.prototype.type`, as `File` interface does not has a property member `type`, it simply inherits the property `type` from `Blob` interface

also some updates to `File` summary of properties

and update api from `*.prototype.*` to `*.*` for `File` interface and `Blob` interface

### Additional details

https://developer.mozilla.org/en-US/docs/Web/API/Blob/type

https://w3c.github.io/FileAPI/#blob-section

image

https://developer.mozilla.org/en-US/docs/Web/API/File/type

https://w3c.github.io/FileAPI/#file-section

image

### CODE REVIEW

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Instead of simply deleting a page, you need to run the command locally: `yarn content delete Web/API/File/type --redirect Web/API/Blob/type`. Also, in web API, we don't use `*.prototype.*`—the link is simply `*.*`.

**Comment by skyclouds2001** [comment]
> Instead of simply deleting a page, you need to run the command locally: `yarn content delete Web/API/File/type --redirect Web/API/Blob/type`. Also, in web API, we don't use `*.prototype.*`—the link is simply `*.*`.

Done

**Review by Josh-Cena — APPROVED** [review]
LGTM, thanks

File: files/en-us/web/api/file/type/index.md [hunk 1]
```diff
@@ -1,69 +0,0 @@
...
-```
-
-### Result
-
-{{EmbedLiveSample('Examples')}}
-
-> **Note:** Based on the current implementation, browsers won't actually read the bytestream of a file to determine its media type.
-> It is assumed based on the file extension; a PNG image file renamed to .txt would give "_text/plain_" and not "_image/png_". Moreover, `file.type` is generally reliable only for common file types like images, HTML documents, audio and video.
-> Uncommon file extensions would return an empty string.
-> Client configuration (for instance, the Windows Registry) may result in unexpected values even for common types. **Developers are advised not to rely on this property as a sole validation scheme.**
```
Comment (Josh-Cena) [inline files/en-us/web/api/file/type/index.md:56]: @skyclouds2001: this information is now lost. Is it still relevant? If so, could you move it to `Blob.type` instead?

Comment (skyclouds2001) [reply files/en-us/web/api/file/type/index.md:56]: Good idea

File: files/en-us/web/api/blob/type/index.md [hunk 2]
```diff
@@ -10,6 +10,11 @@ browser-compat: api.Blob.type
 
 The **`type`** property of a {{domxref("Blob")}} object returns the {{Glossary("MIME type")}} of the file.
 
+> **Note:** Based on the current implementation, browsers won't actually read the bytestream of a file to determine its media type.
+> It is assumed based on the file extension; a PNG image file renamed to .txt would give "_text/plain_" and not "_image/png_". Moreover, `file.type` is generally reliable only for common file types like images, HTML documents, audio and video.
```
Comment (Elchi3) [inline files/en-us/web/api/blob/type/index.md:14]:
```suggestion
> It is assumed based on the file extension; a PNG image file renamed to .txt would give "_text/plain_" and not "_image/png_". Moreover, `blob.type` is generally reliable only for common file types like images, HTML documents, audio and video.
```

**Review by Elchi3 — APPROVED** [review]
Thanks @skyclouds2001 👍

**Pull request merged**